root/account/Register.js

```diff
@@ -78,11 +78,11 @@ component Register(
       <p>
         {exp.l(
           `Follow our {bl|blog} or {tw|twitter account}!
-           To talk to other users, try the {fo|forums} or {irc|IRC}.`,
+           To talk to other users, try the {fo|forums} or the {chat|chat}.`,
```

Comment:
Shouldn't this also say live chat? Or can we just use "chat" everywhere?

Reply:
I don’t have any argument for one or the other.